dockers/docker-dhcp-server/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+if [ "${RUNTIME_OWNER}" == "" ]; then
+    RUNTIME_OWNER="kube"
+fi
+
+CTR_SCRIPT="/usr/share/sonic/scripts/container_startup.py"
```

Comment:
add comments to explain why we need to do this

Reply:
Fixed